assets/js/components/adminbar/AdminBarWidgets.stories.js

```diff
@@ -156,15 +161,43 @@ AnalyticsInactiveNew.storyName = 'Inactive: Analytics New CTA';
 AnalyticsInactiveNew.decorators = [
 	( Story ) => {
 		const setupRegistry = ( registry ) => {
-			// Set up the search console module store but provide no data.
+			provideModules( registry );
+
+			setupSearchConsoleMockReports( registry );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+AnalyticsInactiveNew.parameters = {
+	features: [ 'zeroDataStates' ],
+};
+AnalyticsInactiveNew.scenario = {
+	label: 'Views/AdminBarApp/AdminBarWidgets/AnalyticsInactiveNew',
+	delay: 3000,
+};
+
+export const AnalyticsInactiveNewCompleteActivation = Template.bind( {} );
+AnalyticsInactiveNewCompleteActivation.storyName =
+	'Inactive: Analytics New Complete Activation CTA';
+AnalyticsInactiveNewCompleteActivation.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			// Set up the search console and analytics module store but provide no data.
 			provideModules( registry, [
 				{
-					slug: 'search-console',
+					slug: 'analytics',
 					active: true,
-					connected: true,
+					connected: false,
+					// storeName: 'modules/analytics',
```

Comment:
Is this omission intentional or was this left in by mistake?

Reply:
Oops!